src/Courses/Settings/OverridesForm.tsx

```diff
@@ -36,27 +36,22 @@ export const OverridesForm: React.FC<OverridesFormProps> = ({topic, userId, prob
         }
     );
     const [formLoading, setFormLoading] = useState<boolean>(false);
-    const [submitError, setSubmitError] = useState<string>('');
     const drawerFontSize = '1.4em';
     const { startDate, endDate, deadDate } = watch();
     const [defaultTopic, setDefaultTopic] = useState<NewCourseTopicObj | undefined>(topic);
     const [defaultProblem, setDefaultProblem] = useState<ProblemObject | undefined>(problem);
-    // TODO: This should be provided by react-hook-forms
-    const [ isSubmitSuccessful, setIsSubmitSuccesful] = useState<boolean>(false);
-    const { /* isSubmitSuccessful,*/ isSubmitting  } = formState;
+    const { isSubmitSuccessful, isSubmitting  } = formState;
 
     useEffect(()=>{
         setDefaultTopic(topic);
         setDefaultProblem(problem);
-        setIsSubmitSuccesful(false);
         reset();
     }, [topic, problem, userId]);
 
     // Get Topic Override information
     useEffect(()=>{
         if (!topic || problem !== undefined) return;
         setFormLoading(true);
-        setSubmitError('');
```

Comment:
Are we not clearing the error?
Does the error clear when you move to another override?

Reply:
Yes, the error clears when you move to another override. The logic has been passed to react-hook-forms rather than managing our own state.